msteams-platform/toolkit/toolkit-v4/TeamsFx-preview-and-customize-app-manifest-vs.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: Customize Teams App Manifest in Teams Toolkit using Visual Studio
+title: Customize App Manifest in Teams Toolkit using Visual Studio
 author: surbhigupta
-description: In this module, learn how to edit, preview and customize Teams App Manifest in different environment in Teams Toolkit.
+description: In this module, learn how to edit, preview and customize App Manifest in different environment in Teams Toolkit.
```

Comment:
Could we decapitalize "App Manifest" in title/description for consistency with other Teams docs?

Reply:
Updated.